docs/platform/toc_all.md

```diff
@@ -75,7 +75,8 @@
 ### [Modifying Certificate Manager Texts](/platform/reference/tizen-studio/example-web-certi-ext.md)
 ### [Modifying Eclipse Themes](/platform/reference/tizen-studio/example-web-theme-ext.md)
 
-## [Setting up docker](/platform/reference/setting-up-docker.md)
+## [Tizen Image Creator FAQ](/platform/reference/tic-faq.md)
+## [Set up docker](/platform/reference/set-up-docker.md)
```

Comment:
@asm-foysal, whenever we change the file name we check if the old file name is referenced in any other page and change them as well. @sagarkedari could you help him on how to check on the same.

Reply:
![image](https://user-images.githubusercontent.com/94053723/145809161-2300d508-123e-4e30-8eaa-b12ea7a422a7.png)
@annie-samsung, I discussed this with Sagar and got the hang of checking if a removed/renamed file was referenced in any other pages. It appears the "setting-up-docker" was referenced only on these files so we should be fine.